ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/modals/dmdownload/dmdownload.html

```diff
@@ -0,0 +1,115 @@
+     data-sly-test.downloadAssets="${properties.excludeOriginalAssets ? false : true}"></sly>
+
+<style data-sly-test="${!wccmode.disabled}">
+    .cmp-modal-download .ui.modal {
+        position: relative !important;
+        display: block !important;;
+        top: 0 !important;;
+    }
+</style>
+
+<sly data-sly-test="${wcmmode.edit && !download.isDynamicMediaEnabled}">
+    <p>Note: Dynamic Media is not enabled. Please enable it before configuring the download modal.</p>
+</sly>
+
+<form method="post"
+      action="${download.assets[0].path}.assetdownload.zip/${download.zipFileName}.zip"
+      data-asset-share-id="dm-download-modal"
+      class="ui modal cmp-modal-download--wrapper cmp-modal">
+
+    <input type="hidden" name="_charset_" value="utf-8"/>
+    <input type="hidden" name="licenseCheck" value="true"/>
+    <input type="hidden" name="flatStructure" value="false"/>
+
+    <input type="hidden" name="downloadAssets"
+           value="${downloadAssets ? 'true' : 'false'}"/>
+
+    <i class="close icon"></i>
+
+    <div class="header">
+        ${properties['modalTitle']}
+    </div>
+    <div class="image scrolling content cmp-content">
+        <div class="ui medium image">
+            <div class="ui header">${properties['assetListTitle']}</div>
+
+            <div class="ui list" data-sly-list.asset="${download.assets}">
+                <div class="item">
+                    <i class="file icon"></i>
+                    <div class="content">
+                        <div class="header">${asset.title}</div>
+                    </div>
+                </div>
+            </div>
+
+        </div>
+        <div class="description ui form">
+            <div class="ui header">${properties['downloadOptionsTitle']}</div>
+
+            <sly data-sly-list.asset="${download.assets}">
+                <input type="hidden" name="path"  value="${asset.path}"/>
+            </sly>
+
+            <div class="field">
+                <div class="ui checkbox">
+                    <input type="checkbox" tabindex="3" class="hidden" name="downloadRenditions" value="true"
+                           data-sly-attribute.checked="${!downloadAssets}"
+                           data-sly-attribute.required="${!downloadAssets}"
+                           data-sly-attribute.readonly="${!downloadAssets}">
+                    <label>${"Include Renditions" @ i18n} ${downloadAssets}</label>
+                </div>
+            </div>
+
+            <div class="field">
+                <div class="ui checkbox">
+                    <input type="checkbox" tabindex="4" class="hidden" name="downloadSubassets" value="true">
+                    <label>${"Include Sub-assets" @ i18n}</label>
+                </div>
+            </div>
+
+            <div class="ui selection dropdown search" data-asset-share-id="dm-image-presets" data-sly-test="${download.isDynamicMediaEnabled}">
```

Comment:
See above in JS...

Could we use "real" form elements ...

```
<select class="ui dropdown" data-sly-list.preset="${download.imagePresets}">
    <option value="imagepreset:${preset}">${preset}</div>
</select>
```

and bypass all the JS funky-ness? I think we wouldnt even need a JS file for this modal and we could re-use the"original" download.js ?

Reply:
Yes. I will look at this.